requirements.txt

```diff
@@ -5,6 +5,8 @@ pytest==7.2.1
 pytest-mypy-plugins==1.10.1
 djangorestframework==3.14.0
 types-pytz==2022.7.1.0
+types-requests==2.28.11.8
+types-urllib3==1.26.25.4
```

Comment:
Latest types-requests and types-urllib3 are already incompatible with mypy 0.991.

I think it's a good idea to keep these pinned in any case.

Reply:
Definitely a good idea to pin those dependencies in general